workflow-testing/src/test/java/com/squareup/workflow1/testing/RealRenderTesterTest.kt

```diff
@@ -48,6 +48,9 @@ internal class RealRenderTesterTest {
   private interface OutputWhateverChild : Workflow<Unit, Unit, Unit>
   private interface OutputNothingChild : Workflow<Unit, Nothing, Unit>
 
+  // [WorkflowAction::toString] includes "@-${hashCode()}", so strip it.
+  private fun String.removeActionHashCodes(): String = replace(Regex("-@([0-9]*)"), "")
```

Comment:
Not sure where `toString()` is leaking in, but can you instead change this test to rely on `debugginName` directly and avoid the need for this function? Same for the other instances.

Reply:
This one is a good candidate for that, but not the `TracingWorkflowInterceptorTest` as we want that to use the actual `toString()` or the traces won't be as useful.